### PR TITLE
Ancient map anti alias

### PR DESCRIPTION
**Pull request by MM1nd**
This fails the test as discussed in #250 

The blessed image would have to be changed. It is two full minutes (~66%) faster on a 1024*512 map on my machine though.

### CODE REVIEW

File: worldengine/drawing_functions.py [hunk 1]
```diff
@@ -490,37 +497,17 @@ def on_border(pos):
...
-            r = int(tot_r / n)
-            g = int(tot_g / n)
-            b = int(tot_b / n)
-            target[y, x] = (r, g, b, 255)
-
-        for i in range(steps):
-            _anti_alias_step()
-
-    anti_alias(1)
+    #don't anti-alias the alpha channel
```
Comment (ftomassetti) [inline worldengine/drawing_functions.py]: could we put a space after the sharp symbol?

File: worldengine/drawing_functions.py [hunk 2]
```diff
@@ -490,37 +497,17 @@ def on_border(pos):
...
+    for c in range(num_channels-1):
+        channels[c] = anti_alias_channel(channels[c], 1)
+
+    
+    #hand over to the old implementation
```
Comment (ftomassetti) [inline worldengine/drawing_functions.py]: same here

**Comment by ftomassetti** [comment]
The code looks good to me. I would first merge the changes into the blessed images (PR to worldengine-data) and then rerun the tests here. As they pass I would merge this PR.
Does it make sense?

**Comment by psi29a** [comment]
I agree, if you would like to update the blessed images first, we can compare the differences there too.

**Comment by psi29a** [comment]
Restarting build...

**Comment by psi29a** [comment]
Coveralls is not happy... but the tests pass. :)

**Comment by psi29a** [comment]
Merged, this is great work. Thank you for taking the time!

Please take the time to update the README Contributors section to add your own blurb about yourself. :)

**Pull request merged**